Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fixpoints (0.2.4)
+    fixpoints (0.2.6)
```

Comment:
please only jump one version point

Reply:
you are completely right. `0.2.6` should be the right one. i forgot to update this last time.